kaylaewood/RefactorTractor
# DRYing scripts.js file

#### What's this PR do?
This PR removed redundancy from the scripts.js file. I consolidated all redundant functions into dynamic functions that use arguments and parameters.

#### Where should the reviewer start?
You should only be looking at the scripts.js file. Look specifically at getStatus & getFriendsInfo

#### How should this be manually tested?
I am not sure how to test this, since it's all DOM manipulation. I just looked at the browser view and refreshed many times to make sure that the images, phrases, and charts were updating correctly as the user changed. @brittanystoroz @khalidwilliams @RaHughes, let me know if there is currently a better way to test this!

#### Any background context you want to provide?
N/A

#### What are the relevant tickets?
This affects ticket #2, but does not close it, as I need to continue working through the other files.

#### Screenshots (if appropriate)
N/A

#### Questions:
- Is there a blog post? N/A
- Does the knowledge base need an update? No
- Does this add new dependencies which need to be added? No
